src/main/java/com/Elessar/database/MyDatabase.java

```diff
@@ -0,0 +1,35 @@
+package com.Elessar.database;
+
+import com.Elessar.app.server.User;
+import java.util.List;
+
+/**
+ * Created by Hans on 1/20/19.
+ */
+public interface MyDatabase {
+    String USERS = "users";
```

Comment:
public static final

Reply:
For interface, public static final seems to be the default

Reply:
https://stackoverflow.com/questions/17011374/are-public-and-public-final-redundant-for-interface-fields